src/main/webapp/app/exercises/shared/exercise/exercise.utils.ts

```diff
@@ -196,14 +196,10 @@ export const participationStatus = (exercise: Exercise, testRun?: boolean): Part
     return ParticipationStatus.INACTIVE;
 };
 
-/**
```

Comment:
Comment describes what the code does and not why.
Also the third point was deprecated since that related to an earlier implementation of that function.